content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -626,6 +626,10 @@ We strongly recommend using this admission controller if you intend to make use
 
 The `StorageObjectInUseProtection` plugin adds the `kubernetes.io/pvc-protection` or `kubernetes.io/pv-protection` finalizers to newly created Persistent Volume Claims (PVCs) or Persistent Volumes (PV). In case a user deletes a PVC or PV the PVC or PV is not removed until the finalizer is removed from the PVC or PV by PVC or PV Protection Controller. Refer to the [Storage Object in Use Protection](/docs/concepts/storage/persistent-volumes/#storage-object-in-use-protection) for more detailed information.
 
+### TaintNodesByCondition {#taintnodesbycondition} {{< feature-state for_k8s_version="v1.12" state="beta" >}}
+
+This admission controler taints newly created Node objects as "not ready". This fixes a race condition that could cause pods to be scheduled on new nodes before their taints were updated to accurately reflect their reported conditions.
```

Comment:
Some nits:

This admission controller taints newly created Nodes as "NotReady" with "NoSchedule" effect. This fixes a race condition that could cause Pods to be scheduled on new Nodes before their taints were updated to accurately reflect their reported conditions.

Reply:
@Huang-Wei  - updated.